GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCORPlatform.h

```diff
@@ -156,6 +156,8 @@ id<NSSecureCoding> _Nullable GDTCORDecodeArchive(Class archiveClass,
                                                  NSData *_Nullable archiveData,
                                                  NSError *_Nullable *error);
 
+BOOL GDTCORWriteDataToFile(NSData *data, NSString *filePath, NSError **outError);
```

Comment:
Why is the public API changing?

Reply:
Similar methods were in this file, so placed this one here as well. Can you see any issues with it?

Reply:
It looks like it follows the style. 

I don't understand why the public API surface is so big, but that question is probably out of scope for this PR.

Reply:
Created #6383 to deal with it.